src/silo_api/database_directory_watcher.cpp

```diff
@@ -137,6 +137,11 @@ void silo_api::DatabaseDirectoryWatcher::checkDirectoryForData(Poco::Timer& /*ti
    try {
       database_mutex.setDatabase(silo::Database::loadDatabaseState(most_recent_database_state->first
```

Comment:
I would possibly add a log statement before starting key events as well. That makes it easier to see timings etc, these version changes are rare so logging liberally is fine

Reply:
What do you mean exactly? There is a log statement right before the `try` (in general this class consists of 50% log statements :D )

Reply:
Ah thanks it wasn't showing in default PR view without expanding context